frontend/src/app/restaurant/page.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+interface Restaurant {
+  name: string
+  address: string
+  cuisine: string
+  rating: string
+  phone: string
+  image_url: string
+  website_url: string
+}
+
+interface RestaurantProps {
+  restaurant: Restaurant
+}
+
+interface GetRestaurantsResponse {
+  data: Restaurant[]
+}
+
+const RestaurantCard: React.FC<RestaurantProps> = ({ restaurant }) => {
```

Comment:
Suggest moving `RestaurantCard` and `RestaurantPage` to separate component files, to separate the components from the presentation on this specific page.

I did some googling and it seems like there isn't a single standard for where to store components - but that they shouldn't be stored in the same folder as the routes, because then Next will treat them as routes and it will increase rendering time. Some people have suggested creating a `component` folder either in the project root, or in the `src` folder, for people who use it (like us!)

Plop is already set up to scaffold components in the `src/component` folder, so if there's no strong feelings on the subject, I'd suggest we use that structure. If it ain't broke, don't fix it. 😄